ldk/go/readme.md

```diff
@@ -11,9 +11,76 @@ go get -u github.com/open-olive/loop-development-kit/ldk/go
 Full documentation available on [pkg.go.dev](https://pkg.go.dev/github.com/open-olive/loop-development-kit/ldk/go).
 
 ### Loops
-This LDK can be used to write loops for Olive Helps. More detail about loops is available [here](docs/loops.md).
+Loops receive events and use them to generate relevant whispers. Loops choose which events they want to use and which they want to ignore.
 
-Example loops are available in the examples folder.
+This LDK can be used to write loops for Olive Helps. An example of the loop implementation is available [here](docs/loops.md).
+
+
+#### Interface
+
+Writing a loop boils down to writing an implementation for the `Loop` interface and serving it via `ldk.ServeLoopPlugin`.
+
+```go
+type Loop interface {
```

Comment:
This reads as a part of the code below, and I don't _think_ it should be? Maybe break it from the `main` function below as a separate codeblock, so it'd be like:
```go
type Loop interface {
  LoopStart(ldk.Sidekick) error
  LoopStop() error
}
```

```go
import $LDK_MODULE$

func main() {
  # Instantiate Loop
  # Instantiate Logger
  ldk.ServeLoopPlugin(logger, loop)
}
```

Reply:
Separated.